rabbitmq/datadog_checks/rabbitmq/data/conf.yaml.example

```diff
@@ -30,7 +30,7 @@ instances:
     ## @param rabbit_api_url - string - required
     ## For every instance a 'rabbitmq_api_url' must be provided, pointing to the api
     ## url of the RabbitMQ Managment Plugin (http://www.rabbitmq.com/management.html).
-    #
+    # if https is used and cert chain verification should be skipped, include ssl_verify: False
```

Comment:
There is one extra `#` missing here to match the configuration file norm: https://docs.datadoghq.com/developers/integrations/new_check_howto/#configuration-file

Reply:
yep sorry about that. can add that and use the correct format